composer.json

```diff
@@ -19,6 +19,7 @@
         "doctrine/doctrine-bundle": "^1.12 || ^2",
         "symfony/swiftmailer-bundle": "^2.3 || ^3",
         "sensio/framework-extra-bundle": "^3 || ^4 || ^5 || ^6",
+        "shuchkin/simplexlsxgen": "^1.2"
```

Comment:
you're missing a comma here which would render the json invalid. And the current version of the library is 1.3, the API stayed the same so I'd say it's safe to just update it